terra_oracle_vote.py

```diff
@@ -71,6 +73,7 @@
 alertmisses = os.getenv("MISS_ALERTS", "true") == "true"
 debug = os.getenv("DEBUG", "false") == "true"
 metrics_port = os.getenv("METRICS_PORT", "19000")
+band_endpoint = os.getenv("BAND_ENDPOINT", "https://poa-api.bandchain.org")
```

Comment:
I would also like to add this to oracle.server and readme.md as well.

Reply:
done